src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import {action, observable} from 'mobx';
+import React from 'react';
+import {translate} from '../../../utils/Translator';
+import Overlay from '../../../components/Overlay';
+import Form from '../../../components/Form';
+import SingleSelect from '../../../components/SingleSelect';
+import resourceEndpointRegistry from '../../../services/ResourceRequester/registries/ResourceRouteRegistry';
```

Comment:
That's called `resourceRouteRegistry` now, please also rename the variable accordingly.